servers/cromwell/jobs/__main__.py

```diff
@@ -67,6 +71,13 @@
 try:
     with open(capabilities_path) as f:
         capabilities_config = json.load(f)
+    for settings in capabilities_config['displayFields']:
+        if settings['field'].startswith(LABELS_PREFIX):
+            if len(settings['field']) == len(LABELS_PREFIX) or len(
```

Comment:
Is there any way we could define the `minLength` and `maxLength` in the swagger definition of the model so that we don't need to do the validation here?

Reply:
The restrictions can be outlined in swagger, but that means that the problem won't be visible until a user tries to change something relating to that label (the request will fail and user sees an error in the snackbar "Invalid label: ...") via the UI. I think, when it's a configuration problem, it's best to fail when the application is being started up, since the person doing so is more likely to know how to fix that sort of issue (as opposed to a user).  See the discussion here: https://elastc.com/c/jsfz&&H7/304-label-user-input-validation.

Reply:
Oh that makes sense!